### PR TITLE
fix thumbnails squashed

### LINKED ISSUE #374: thumbnails on testing are squashed
fix aspect ratio

child of #457

### PR DESCRIPTION
**Pull request by boney-bun**
fix #374

### CODE REVIEW

File: geonode/qgis_server/helpers.py [hunk 1]
```diff
@@ -409,8 +409,8 @@ def thumbnail_url(bbox, layers, qgis_project, style=None, internal=True):
         'REQUEST': 'GetMap',
         'BBOX': bbox,
         'SRS': 'EPSG:4326',
-        'WIDTH': '250',
-        'HEIGHT': '250',
+        'WIDTH': '240',
+        'HEIGHT': '180',
```
Comment (lucernae) [inline geonode/qgis_server/helpers.py]: @boney-bun can you first explain why it has to be this value?
IMO, we should find a way to request a square thumbnail, then the aspect ratio can be tweak in the UI using CSS.

Comment (boney-bun) [reply geonode/qgis_server/helpers.py]: @lucernae the numbers is coming from the geoserver. i tried to imitate what the geoserver do with the thumbnail as in https://github.com/kartoza/geonode/blob/master-qgis_server/geonode/geoserver/helpers.py#L1651

**Comment by gubuntu** [comment]
We do not want to request a square thumbnail and then change its access ratio -- it looks terrible (and incorrect). 

We want to request a thumbnail in the same aspect ratio and the same resolution as the style of the div it will fill, so it can slot straight it. Thumbnail sizes should be consistent across GeoNode so use whatever the convention is in GeoNode.

**Comment by lucernae** [comment]
I will close this, some part of the fixes were done in #464.
The rest is to fix the requested extent as a square.

> We do not want to request a square thumbnail and then change its access ratio -- it looks terrible (and incorrect).We want to request a thumbnail in the same aspect ratio and the same resolution as the style of the div it will fill, so it can slot straight it. Thumbnail sizes should be consistent across GeoNode so use whatever the convention is in GeoNode.

Just some clarification, the div slot have auto height, so it is best to use a square thumbnail, since the width will have maximum width and the height will follow. GeoServer's thumbnail also looks like this because it's not square. 

At the moment, the aspect ratio is fixed, but if the requested extent is not square (for example, like the length of Java Island), it will appear very short even though it will not look squashed. I was thinking we should add a small UI to select the extent and toggle the basemap, in case the user wants to have different size.

**Pull request closed without merging**